doc/integrator/docker.rst

```diff
@@ -48,6 +48,11 @@ Docker does not clean anything automatically, in particular it does not clean an
 therefore disk space may become problematic after a certain number of builds.
 You can use the following commands to manually remove Docker files.
 
+If you use Docker version (``docker --version``) >= ``1.13`` use ``docker system prune`` to do a clean,
+you can add an ``--all`` option to do a deeper clean.
+
```

Comment:
Seems clear to me  - small suggestion for readability:

If you use Docker version (``docker --version``) >= ``1.13`,` use ``docker system prune`` to clean files; you can add the ``--all`` option to do a deeper clean.